src/Autumn/aprograms/particles.jl

```diff
@@ -1,3 +1,87 @@
+using Distributions
+
+""" ----- custom types ----- """
+struct Position
+  x::Int
+  y::Int
+end
+
+struct Click
+  x::Int
+  y::Int
+end
+
+struct Particle
+  position::Position
+end
+
+""" ----- global variables --- """
+GRID_SIZE = 16
+particles = []
+time = 0
+
+# history-related 
+particlesHistory = Dict{Int64, Any}
```

Comment:
Dict{Int, Any} will lead to loss of type information as I talked about last week